plugin-locationcomponent/src/main/java/com/mapbox/maps/plugin/locationcomponent/PermissionsManager.java

```diff
@@ -0,0 +1,182 @@
+package com.mapbox.maps.plugin.locationcomponent;
+
+import android.Manifest;
+import android.app.Activity;
+import android.content.Context;
+import android.content.pm.PackageInfo;
+import android.content.pm.PackageManager;
+import android.os.Build;
+import android.util.Log;
+
+import androidx.core.app.ActivityCompat;
+import androidx.core.content.ContextCompat;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * Helps request permissions at runtime.
+ */
+public class PermissionsManager {
+
+    private static final String LOG_TAG = "PermissionsManager";
+    private static final String COARSE_LOCATION_PERMISSION = Manifest.permission.ACCESS_COARSE_LOCATION;
+    private static final String FINE_LOCATION_PERMISSION = Manifest.permission.ACCESS_FINE_LOCATION;
```

Comment:
Since they're already constants - is there any benefit to create our own? Maybe just inline `Manifest.permission.ACCESS_FINE_LOCATION` ?

Reply:
Ah, this file seems to be copied from other lib, nevermind then